index.html

```diff
@@ -30,5 +33,126 @@ <h2>I'm a Full Stack Developer</h2>
             </div>      
         </section>
       </header>
+
+<!-- Here main content starts -->
+      <main>
+        <section class="works">
+          <div class="title">
+            <h2>My Recent Works</h2>
+            <hr>
+          </div>
+          <div class="daily-selection">
+            <div class="img-placeholder">
+                <a href="https://www.freecodecamp.org/certification/fccde5c1dbd-ac5b-4ee7-91c7-386adbe10dcd/responsive-web-design"><img src="Projects/Project_1.png" alt=""></a>
+            </div>
+            <article>
+                <h3>Multi-Post Stories</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            </article>
+            <ul class="tags">
+                <li>html</li>
+                <li>css</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+            <div class="see-more-container">
+                <a class="btn see-more-btn" href="https://www.freecodecamp.org/certification/fccde5c1dbd-ac5b-4ee7-91c7-386adbe10dcd/responsive-web-design" target="_blank" rel="noopener">See more</a>
+            </div>
+            
+          </div>
+
+          <div class="projects-grid">
+
+            <div class="project-card card-one">
+                <h3 class="card-title">Personal Portfolio Carrera</h3>
+                <p class="card-description">A personal portfolio to show different kinds of projects. This portfolio is the connection between the developer and the final customer or recruiter.</p>
+                <ul class="card-tags">
+                    <li>html</li>
+                    <li>css</li>
+                </ul>
+                 <a class="btn link-card-button" href="https://carreraprogrammer.github.io/carreraprogrammerportfolio/" target="_blank" rel="noopener">See Project</a>
+                
+            </div>
```

Comment:
- [ ] Kindly make sure your card looks like in the Figma design. Add margins (top, bottom, left, right) defined in the design as defined in the design and please make sure you use the right card you can even download it from Figma. You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma]
## Figma design

![image](https://user-images.githubusercontent.com/95297251/185692372-3a999b46-ef6e-459f-bbea-1a4fffe023f2.png)
![image](https://user-images.githubusercontent.com/95297251/185692607-bdf5b5a3-c598-41f9-accf-3a7889e8304c.png)


## your design
![image](https://user-images.githubusercontent.com/95297251/185692264-e2c7d493-2d8a-4a37-b582-ee9f2a20455e.png)
![image](https://user-images.githubusercontent.com/95297251/185692713-5bc7578c-6cdc-4e20-ab3e-36ab9a0fa937.png)